locale/de_DE.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -131,6 +131,18 @@ msgstr "Hilfe + Extras"
 msgid "Team display"
 msgstr "Team-Monitor"
 
+#: index.php:1090
```

Comment:
These additions look strange because there are no corresponding additions for the other languages.